system-monitor-graph@rcassani/files/system-monitor-graph@rcassani/desklet.js

```diff
@@ -120,7 +120,7 @@ SystemMonitorGraph.prototype = {
         }
 
         // Desklet proportions
-        let unit_size = 15 * this.scale_size;  // pixels
+        let unit_size = 15 * this.scale_size * global.ui_scale;  // pixels
```

Comment:
`unit_size` does not need modification. 
The size of the graph (in pixels) should remain the same regardless `global.ui_scale`. Only text size needs to be scaled.

Reply:
I think it would be better to take `global.ui_scale` into account for `unit_size` because otherwise the desklet will appear unexpectedly small by default on HiDPI screens.

Reply:
Yes, I'll look small although that is the expected behaviour. The `Scale factor` in the Desklet settings indicates that `1 = 330×120 px`